frontend/public/locales/zh/translation.json

```diff
@@ -65,6 +65,7 @@
   "Kiali home cluster": "Kiali home cluster",
   "Kiali home cluster: {{name}}": "Kiali主集群: {{name}}",
   "Kiali on GitHub": "Kiali on GitHub",
+  "Kubernetes": "Kubernetes",
```

Comment:
I guess this maybe has a translation into other character sets?  I don't know, not a problem to include it even if they leave it alone.

Reply:
I don't know, probably there is no translation. I just changed `Version` to `Kubernetes` and I did not care about the translation. Anyway, if that is the case we can remove it afterwards.